hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFsSnapshot.java

```diff
@@ -63,7 +63,7 @@ public class TestOzoneFsSnapshot {
   private static OzoneManager ozoneManager;
   private static OzoneFsShell shell;
   private static final String VOLUME =
-      "vol-" + RandomStringUtils.randomNumeric(5);;
+      "vol-" + RandomStringUtils.randomNumeric(5);
```

Comment:
This file was included in the latest commit.  Looks like unintended.  Please revert this addition.